x-pack/plugins/ingest_manager/server/plugin.ts

```diff
@@ -78,18 +66,15 @@ const allSavedObjectTypes = [
   ENROLLMENT_API_KEYS_SAVED_OBJECT_TYPE,
 ];
 
-export class IngestManagerPlugin implements Plugin<IngestManagerSetupContract> {
+export class IngestManagerPlugin implements Plugin<void, IngestManagerStartupContract> {
```

Comment:
I believe `IngestManagerSetupDeps` is the third parameter

Reply:
@jfsiii It was not there before, but I am surprised it worked without it. But I can definitely add it. please let us talk when you are fully online so we get on the same page.